### DISCUSSION TITLE
Spanish lemmatization suffixes

### DISCUSSION THREAD

**Discussion by nachouve** [explosion/spaCy#6601] [Language Support]
## How to reproduce the behaviour

I've found that some suffixes are not managed properly in spanish. Suffixes "-azo" and "-ísimo" are ignored.

```
import spacy
nlp = spacy.load('es_core_news_md')
doc = nlp('golpazo finísimo')

for i in doc:
    print(i, i.lemma_, i.pos_)
```

> Prints:
> golpazo golpazo NOUN
> finísimo finísimo ADJ

Lemmas should be "golpe" and "fino".

Maybe there are more suffixes to check. Here there are some useful resources:
[https://espanol.lingolia.com/es/vocabulario/formacion/sufijos](url) 
[https://www.rae.es/sites/default/files/Elementos_compositivos_prefijos_y_sufijos_del_espanol_Esencial.pdf](url)

## Your Environment

* Operating System: Windows 10
* Python Version Used: 3.8
* spaCy Version Used: 2.3.5

**Comment by svlandeg** [comment]
(FYI - I converted this issue to a discussion, as we try to keep the issues focused mainly on actual bugs. The forum is the best place to find other collaborators who speak a specific language and perhaps collaborate on improving the language data in spaCy. We very much appreciate community contributions for different languages!)
